cluster/etcd/stack.yaml

```diff
@@ -94,9 +94,6 @@ Resources:
         - Key: StackVersion
           PropagateAtLaunch: true
           Value: etcd
-        - Key: kubernetes:component
-          PropagateAtLaunch: true
-          Value: etcd-cluster
```

Comment:
I'm not 100% sure if we depend on this somewhere, maybe in zmon checks 🤔

Reply:
Alright. I'll restore the tag then.